packages/cosmwasm-typescript-gen/.vscode/settings.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
Does it make sense to delete and gitignore both `packages/cosmwasm-typescript-gen/.vscode` folder and `packages/wasm-ast-types/.vscode` ?